prediction_market_agent/tools/anvil/anvil_requests.py

```
@@ -0,0 +1,56 @@
+            Web3.to_checksum_address(address),
+            {"blockNumber": hex(block)},
+        ],
+        "id": 1,
+    }
+
+    response = requests.post(rpc_url, json=data)
+    try:
+        # response is always 200, hence not raising status
+        balance = HexBytes(response.json()["result"])
+        return block, balance.as_int()
+    except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)